extensions/amp-youtube/1.0/base-element.js

```diff
@@ -25,6 +25,7 @@ BaseElement['Component'] = Youtube;
 BaseElement['props'] = {
   'autoplay': {attr: 'autoplay', type: 'boolean'},
   'loop': {attr: 'loop', type: 'boolean'},
+  'children': {},
```

Comment:
I'm not entirely show what amp-youtube was doing before to get `usesShadowDom == true` but I've added this here because the unit test started failing without it.